crates/wash-lib/src/start/github.rs

```diff
@@ -87,3 +90,41 @@ where
         .await
         .map_or(false, |m| m.is_file())
 }
+
+/// Helper function to set up a reqwest client for performing the download
+fn get_download_client() -> reqwest::Client {
+    let proxy_username = std::env::var("WASH_PROXY_USERNAME").unwrap_or_default();
+    let proxy_password = std::env::var("WASH_PROXY_PASSWORD").unwrap_or_default();
+    // TODO(joonas): should we consider system proxy configuration here, or just explicit config?
```

Comment:
Is this still relevant? specifying `http_proxy` is pretty common for specifying configuration of a proxy, but not sure which system proxy configuration you're referring to here

Reply:
it's probably not relevant here for this particular use case, for context [reqwest](https://github.com/seanmonstar/reqwest/blob/master/src/proxy.rs#L284-L294) has this way of detecting what proxies might be set up at the system level, but I think given the intent of this feature, we probably don't actually care about that since we're expecting that an explicit `HTTP(S)_PROXY` is configured here.

I'll go ahead and remove the TODO, that's a good callout.